cmd/alertmanager/main.go

```diff
@@ -117,7 +117,13 @@ func main() {
 	}
 
 	logger := log.NewLogger(os.Stderr)
-	mrouter := initMesh(*meshListen, *hwaddr, *nickname, *password)
+
+	var mrouter *mesh.Router
+	if *meshListen == "" {
+		mrouter = initMesh(*flag.String("flag-name", "0.0.0.0:0", "flag-msg"), *hwaddr, *nickname, *password)
```

Comment:
With the conditional options used now, we should no longer need to instantiate this dummy router.